docs-ui/components/panels.stories.js

```diff
@@ -24,6 +30,25 @@ storiesOf('UI|Panels', module)
       </Panel>
     ))
   )
+  .add(
+    'Table Panel',
+    withInfo({
+      text: 'A Panel for "tabular" data',
+    })(() => (
+      <PanelTable columns={3}>
+        <div>Header #1</div>
+        <div>Header #2</div>
+        <div>Header #900000</div>
```

Comment:
Would having explicit header elements be a simpler to understand API? and allow the header items to be customized if necessary.

Reply:
Hmmm, I agree this API is a bit weird because we're just assuming the first `n` elements are headers.  What about, instead of `columns`, we have `headers` that accept a `ReactNode`

Reply:
That could work I was proposing 

```jsx
<PanelTable columns={3}>
  <PanelTableHeader>Header #1</PanelTableHeader>
  <PanelTableHeader>Header #2</PanelTableHeader>
  <PanelTableHeader>Header #3</PanelTableHeader>
  <div>Cell 1</div>
  <div>Cell 2</div>
  <div>Cell 3</div>
</PanelTable>

Reply:
Yeah, this would be more flexible, but I'm not sure if that's needed -- outside of padding (e.g. adding a button or input), we don't really change the headers much.